frodo.js

```diff
@@ -1,59 +1,62 @@
-const formattedDate = new Date().toLocaleDateString();
```

Comment:
SCRIPT-This function might benefit from using async/await instead of callbacks for better readability.